topics/ecology/tutorials/PAMPA-toolsuite-tutorial/tutorial.md

```diff
@@ -0,0 +1,847 @@
+ - Supplementary input for the tools to compute a GLM(M) analysis: Observation unit data file containing at least
+   ```observation.unit``` and ```year```. It might also contain ```habitat```, ```site``` and any other information
+   about the location at a given year that seems useful (examples: ```temperature```,```substrate type```,```plant cover```)
+
+> ### {% icon details %} What is the 'observation.unit' field?
+> 
+> The 'observation.unit' field is a primary key linking the observation data file and the metrics data file to an observation 
+> unit data file. An unique 'observation.unit' identifier represents an unique sampling event at a given year and location.
+> Such a file permits to reduce the size of data files, if all informations about the sampling event (examples: 
+> ```temperature```,```substrate type```,```plant cover```) are gathered in the same dataframe as the observations data 
+> (```species sampled```,```count```) same informations will be unnecessarily repeated in many lines.
+> 
+{: .details}
+
+> ### {% icon tip %} The 'observation.unit' nomenclature
+> 
+> For this workflow, two nomenclatures are advised :
+> * The PAMPA file nomenclature: 
+>   ```[Two first letters of 'site'][Two last numbers of 'year'][Identifier of the precise sampled location in four numbers]```
+>   The fields ```year``` and ```location``` will be automatically infered from this nomenclature in the first tools to 
+>   compute metrics
+> 
+> * The automatic nomenclature: ```['year' in four numbers]_[Identifier of the precise sampled location in any character chain]```
+>   This nomenclature will be automatically constructed in the first tools to compute metrics if the input file doesn't already 
+>   have a ```observation.unit``` field but only ```year``` and ```location```
+> 
+> If you already have your own nomenclature for the primary key linking your files, it will remain untouched by the tools but the 
+> output data table from the first tools to compute metrics won't contain ```year``` and ```location``` field.
+> In this case and if you want to add such informations to your metrics data file, you can use the tool **Join** {% icon tool %} 
+> with the metrics data file and the observation unit data file.
+> 
+{: .tip}
+
+> ### {% icon details %} The difference between fields ```site``` and ```location```?
+> 
+> Those two fields both inform a geographical indication. 
+> ```location``` is the finest geographical level and ```site``` may represent a wider geographical range but can also 
+> represent the same geographical level as ```location```. This choice is up to your discretion, depending which geographical
+> level you want to take into account in your statistical analysis.
+> 
+{: .details}
+
+### Concatenation of data files
+
+> ### {% icon hands_on %} Hands-on: Data files concatenation
+>
+> 1. **Concatenate datasets tail-to-head (cat)** {% icon tool %} select the three datasets in {% icon param-files %} 
```

Comment:
```suggestion
> 1. {% tool [Concatenate datasets tail-to-head](Cat1) %} select the three datasets in {% icon param-files %} 
```

there is some new syntax you can use here that will be quite nice for end users. You can read more about it here: https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links